public/locales/ar/terms.json

```diff
@@ -0,0 +1,8 @@
+{
+    "head":"תנאי השימוש",
```

Comment:
Do we want to translate it?

Reply:
I have received it like that after translation, I guess there's not literal translation to this in arabic but can recheck